packages/composite-checkout/package.json

```
@@ -48,13 +48,16 @@
 		"react-stripe-elements": "^4.0.2"
 	},
 	"devDependencies": {
-		"react": "^16.12.0",
-		"react-dom": "^16.12.0",
 		"@automattic/calypso-build": "^7.0.0",
 		"@automattic/calypso-polyfills": "^1.0.0",
 		"@testing-library/jest-dom": "^5.9.0",
 		"@testing-library/react": "^10.0.5",
+		"jest": ">=26.4.0",
```

Using `"jest": "^26.4.0"` is consistent with the rest of the repo and satisfies the required peer dependency.
```suggestion
		"jest": "^26.4.0",
```